### DISCUSSION TITLE
Connection with InetlliJ IDEA fail

### DISCUSSION THREAD

**Discussion by McBlockYT** [community/community#133515] [New to GitHub]
Hi there,
I'm just trying to connect my github account with Inellij to backup my projects, but I always get an error when trying to create a new repo via IntelliJ. `Can not load information from github.com/McBlockYT:Request response: Access to this site has been restricted.`

**Comment by KuyaMecky** [comment]
It sounds like you’re having trouble connecting IntelliJ IDEA to GitHub due to access issues. Here are some steps you can follow to troubleshoot and resolve this issue:

1. **Check GitHub Access**: Ensure that you have access to the GitHub repository and that the URL is correct. You can manually visit the URL in your browser to confirm that it’s accessible.

2. **GitHub Permissions**: Verify that your GitHub account has the necessary permissions to create repositories. Sometimes, organizational policies or restrictions might affect access.

3. **Authentication Issues**: If your access to GitHub is restricted, it could be related to authentication issues. Check your GitHub credentials and make sure they are correctly configured in IntelliJ IDEA. You might need to reauthenticate or update your GitHub token.

4. **Proxy Settings**: If you’re behind a proxy or firewall, this could restrict access to external sites like GitHub. Check your network settings and configure your proxy settings in IntelliJ IDEA if needed.

5. **IntelliJ IDEA Configuration**: Make sure you’ve correctly set up GitHub integration in IntelliJ IDEA. Here’s a brief guide to configure it:
   - Go to `File` > `Settings` (or `IntelliJ IDEA` > `Preferences` on macOS).
   - Navigate to `Version Control` > `GitHub`.
   - Add your GitHub account if it’s not already configured.

6. **Update IntelliJ IDEA**: Ensure you’re using the latest version of IntelliJ IDEA. Sometimes, updates include fixes for integration issues.

7. **GitHub Status**: Check GitHub's status page to ensure there are no outages or issues affecting access.

8. **Error Details**: If possible, get more details about the error message or check IntelliJ IDEA’s log files for additional information. The logs might provide more context about what’s going wrong.

If these steps don’t resolve the issue, you might want to contact GitHub support or IntelliJ IDEA support for further assistance.

**Reply by McBlockYT** [reply]
Thanks for the fast reply. But where can I find the IntelliJ logs?

**Reply by KuyaMecky** [reply]
You can find the IntelliJ IDEA logs by following these steps:

1. **Open IntelliJ IDEA**.
2. **Go to Help** in the top menu bar.
3. **Select Show Log in Explorer** (Windows) or **Show Log in Finder** (macOS).

   - On **Windows**: This will open the log directory in File Explorer.
   - On **macOS**: This will open the log directory in Finder.

The log files are typically located in the following directory:

- **Windows**: `C:\Users\<YourUsername>\.IntelliJIDEA<version>\system\log`
- **macOS**: `~/Library/Logs/IntelliJIDEA<version>`

In the log directory, you can find files like `idea.log`, which contain detailed information about the operations and any errors encountered by IntelliJ IDEA.

You can open these log files with any text editor to review the error details related to your issue.
